src/block-components/block-div/edit.js

```diff
@@ -71,7 +71,7 @@ export const Edit = props => {
 					onChange={ hasBackground => setAttributes( { hasBackground } ) }
 					initialOpen={ initialOpen === 'background' }
 				>
-					<BackgroundControls attrNameTemplate="block%s" />
+					<BackgroundControls attrNameTemplate="block%s" isBlockBackground={ true } />
```

Comment:
While this works, I'm thinking that ideally `BackgroundControls` should have no knowledge of an attribute named `hasBackground`. Also, because `isBlockBackground` isn't reusable because it's coupled tightly with `hasBackground`.

What if instead of a `isBlockBackground={ true }` prop which will enable `hasBackground` every time a background color/image is modified inside the component.. we add a `onBackgroundEnableAttribute` that would specify what attribute would be made `true` when the background color/image is modified:

```suggestion
					<BackgroundControls 
						attrNameTemplate="block%s"
						onBackgroundEnableAttribute="hasBackground"
					/>
```